weasyprint/css/utils.py

```diff
@@ -740,3 +740,7 @@ def get_content_list_token(token, base_url):
         elif arg.type == 'string':
             string = arg.value
         return ('leader()', ('string', string))
+    elif name == 'element':
+        if len(args) != 1 or args[0].type != 'ident':
+            return
+        return ('element()', ('element', args[0]))
```

Comment:
the second element of the tpuble could be just `args[0]` I think

Reply:
It can, as `element()` only accepts idents.